core/block/export/export.go

```diff
@@ -451,6 +526,40 @@ func (e *export) handleFileObject(ctx context.Context, keys *files.FileKeys, spa
 	return keys, nil
 }
 
+func (e *export) saveFiles(ctx context.Context, b sb.SmartBlock, wr writer) {
+	st := b.NewState()
+	fileHashes := e.getFileHashes(st)
+	for _, fh := range fileHashes {
+		if _, werr := e.saveFile(ctx, wr, fh); werr != nil {
+			log.With("hash", fh).Warnf("can't save file: %v", werr)
+		}
+	}
+}
+
+func (e *export) getFilesFromRelations(st *state.State, fileHashes []string) []string {
+	for _, relLink := range st.GetRelationLinks() {
+		if relLink.Format == model.RelationFormat_file {
+			if e.isCoverRelationColor(st, relLink) {
```

Comment:
coverId has RelationFormat_longtext, you should check it independent of file relation format

Reply:
Fixed